package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "purecss",
-  "version": "0.6.1-pre",
+  "name": "purecss-conversionready",
```

Comment:
Not at all, this is my personal work in progress branch that I use for clients. We're still working on transitioning the project to new maintainership with yahoo guys.